### PR TITLE
Store each test run in a folder named with a timestamp.

### PR DESCRIPTION
**Pull request by joshcartme**

### CODE REVIEW

**Comment by joshcartme** [comment]
I couldn't figure out how to get mkdirp to include a colon in the pathname, IMO would be much better for the time portion of the timestamp.

**Comment by Mortimerp9** [comment]
A column in the dirname? Heresy!

**Comment by joshcartme** [comment]
Haha, I was thinking more along the lines of a colon : 
So the directory name would end up being something like 2015-06-08_16:45:33

My think is that makes it obvious it is a date and a time.

File: screenshots.js [hunk 1]
```diff
@@ -118,6 +126,19 @@ function QF2E(tests, opts) {
...
+  tests.outpath += [
+    rightNow.getFullYear(),
+    pad(rightNow.getMonth()+1, 2),
+    pad(rightNow.getDate(), 2)
+  ].join('-') + '_' + [
+    pad(rightNow.getHours(), 2),
+    pad(rightNow.getMinutes(), 2),
+    pad(rightNow.getSeconds(), 2)
+  ].join('-') + '/';
+  console.log(tests.outpath)
```
Comment (Mortimerp9) [inline screenshots.js]: no need to commit the log

Comment (joshcartme) [inline screenshots.js]: Just removed it

**Pull request merged**